component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1522,4 +1533,36 @@ private void setUserClaimValues(AuthenticatedUser authenticatedUser, Map<String,
             throw new AuthenticationFailedException(errorMessage, e);
         }
     }
+
+    private long getUnlockTimeInMilliSeconds(AuthenticatedUser authenticatedUser) throws AuthenticationFailedException {
```

Comment:
Shall we add a method comment with the behavior of the method? Why we return `0` likewise? WDYT?

Reply:
fixed in 0cf5704